[DI-6248] Update event documentation (#7491)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events.md

    "app_group_id" : "(optional, string) API ID of the app group this user belongs to",
    "subscription_group_id" : "(optional, string) Subscription group API ID",
    "campaign_id" : "(optional, string) API ID of the campaign this event belongs to",
    "message_variation_id" : "(optional, string) API ID of the message variation this user received",
    "canvas_id" : "(optional, string) API ID of the Canvas this event belongs to",
    "canvas_variation_id" : "(optional, string) API ID of the Canvas variation this event belongs to",
    "canvas_step_id" : "(optional, string) API ID of the Canvas step this event belongs to",
    "dispatch_id" : "(optional, string) ID of the dispatch this message belongs to",
    "provider_error_code" : "(required, string) Error code from WhatsApp",
    "provider_error_title" : "(required, string) Description of failure from WhatsApp",
    "token" : "(required, string) The Mixpanel API token."
  }
}
```
{% endtab %}

{% tab Cloud Storage (S3/Azure Blob/Google Cloud Storage) %}
```json
// WhatsApp Failure: users.messages.whatsapp.Failure

{

>   "app_group_id" : "(optional, string) API ID of the app group this user belongs to",
>   "campaign_id" : "(optional, string) API ID of the campaign this event belongs to",
>   "canvas_id" : "(optional, string) API ID of the Canvas this event belongs to",
>   "canvas_step_id" : "(optional, string) API ID of the Canvas step this event belongs to",
>   "canvas_step_message_variation_id" : "(optional, string) API ID of the Canvas step message variation this user received",
>   "canvas_variation_id" : "(optional, string) API ID of the Canvas variation this event belongs to",
>   "device_id" : "(optional, string) ID of the device on which the event occurred",
>   "dispatch_id" : "(optional, string) ID of the dispatch this message belongs to",
>   "external_user_id" : "(optional, string) External user ID of the user",
>   "from_phone_number" : "(required, string) Phone number used to send in e164 format (for example +14155552671)",
>   "id" : "(required, string) Globally unique ID for this event",
>   "message_variation_id" : "(optional, string) API ID of the message variation this user received",
>   "provider_error_code" : "(required, string) Error code from WhatsApp",
>   "provider_error_title" : "(required, string) Description of failure from WhatsApp",
>   "subscription_group_id" : "(optional, string) Subscription group API ID",
>   "time" : "(required, long) Unix timestamp at which the event happened",
>   "timezone" : "(optional, string) Time zone of the user",
>   "to_phone_number" : "(optional, string) Phone number of the user receiving the message in e164 format (for example +14155552671)",
>   "user_id" : "(required, string) BSON ID of the user that performed this event"

Reviewer: ```suggestion
  "app_group_id" : "(optional, string) API ID of the app group this user belongs to",
  "campaign_id" : "(optional, string) API ID of the campaign this event belongs to",
  "canvas_id" : "(optional, string) API ID of the Canvas this event belongs to",
  "canvas_step_id" : "(optional, string) API ID of the Canvas step this event belongs to",
  "canvas_step_message_variation_id" : "(optional, string) API ID of the Canvas step message variation this user received",
  "canvas_variation_id" : "(optional, string) API ID of the Canvas variation this event belongs to",
  "device_id" : "(optional, string) ID of the device on which the event occurred",
  "dispatch_id" : "(optional, string) ID of the dispatch this message belongs to",
  "external_user_id" : "(optional, string) External ID of the user",
  "from_phone_number" : "(required, string) Phone number used to send in e.164 format (for example +14155552671)",
  "id" : "(required, string) Globally unique ID for this event",
  "message_variation_id" : "(optional, string) API ID of the message variation this user received",
  "provider_error_code" : "(required, string) Error code from WhatsApp",
  "provider_error_title" : "(required, string) Description of the error from WhatsApp",
  "subscription_group_id" : "(optional, string) Subscription group API ID",
  "time" : "(required, long) Unix timestamp at which the event happened",
  "timezone" : "(optional, string) Time zone of the user",
  "to_phone_number" : "(optional, string) Phone number of the user receiving the message in e.164 format (for example +14155552671)",
  "user_id" : "(required, string) BSON ID of the user who performed this event"
```

    "app_group_id" : "(optional, string) API ID of the app group this user belongs to",
    "subscription_group_id" : "(optional, string) Subscription group API ID",
    "campaign_id" : "(optional, string) API ID of the campaign this event belongs to",
    "message_variation_id" : "(optional, string) API ID of the message variation this user received",
    "canvas_id" : "(optional, string) API ID of the Canvas this event belongs to",
    "canvas_variation_id" : "(optional, string) API ID of the Canvas variation this event belongs to",
    "canvas_step_id" : "(optional, string) API ID of the Canvas step this event belongs to",
    "dispatch_id" : "(optional, string) ID of the dispatch this message belongs to",
    "message_extras" : "(optional, string) A JSON string of the tagged key-value pairs during liquid rendering"
  },
  "timestamp" : "(required, long) Unix timestamp at which the event happened",
  "type" : "track",
  "userId" : "(required, string) External user ID of the user"
}
```
{% endtab %}
{% endtabs %}

{% endapi %}

{% api %}

## WhatsApp abort message events

{% apitags %}
WhatsApp, Abort
{% endapitags %}

> This event occurs if a WhatsApp message was aborted based on Liquid aborts, Quiet Hours, etc.

Reviewer: ```suggestion
This event occurs if a WhatsApp message is aborted based on Liquid aborts or Quiet Hours.
```
